src/gluonts/ev/__init__.py

```diff
@@ -114,6 +118,8 @@
     "Aggregation",
     "Sum",
     "Mean",
+    "SumNumMaskedTargetValues",
+    "sum_num_masked_target_values",
```

Comment:
@jaheba I think this namespace is now becoming very confused: it's aggregating stuff from `ev.stats` and `ev.metrics`, and in this example now `num_masked_target_values` (function) is a very different thing from `sum_num_masked_target_values` (a `MetricDefinition` object).

I think we should reconsider what names are exposed: maybe just the ones from `ev.metric`?

Reply:
I guess there is an argument not to expose any of these?

This:

```py
from gluonts.ev.metrics import MASE
from gluonts.ev.stats import absolute_percentage_error
```

looks cleaner to me than:

```py
from gluonts.ev import MASE, absolute_percentage_error
```

Reply:
Maybe; let's take this to a separate story I guess